pennylane/gradients/vjp.py

```diff
@@ -145,18 +144,18 @@ def vjp(tape, dy, gradient_fn, gradient_kwargs=None):
     Executing the VJP tapes, and applying the processing function:
 
     >>> dev = qml.device("default.qubit", wires=2)
-    >>> vjp = fn([t.execute(dev) for t in vjp_tapes])
+    >>> vjp = fn(qml.execute(vjp_tapes, dev, gradient_fn=qml.gradients.param_shift, interface="torch"))
     >>> vjp
-    tensor([-0.6069, -0.0451,  0.0451, -0.0139, -0.2809,  0.2809],
-           dtype=torch.float64, grad_fn=<ViewBackward>)
+    tensor([-1.1562e-01, -1.3862e-02, -9.0841e-03, -1.3878e-16, -4.8217e-01,
+             2.1329e-17], dtype=torch.float64, grad_fn=<ViewBackward>)
 
     The output VJP is also differentiable with respect to the tape parameters:
 
     >>> cost = torch.sum(vjp)
     >>> cost.backward()
     >>> x.grad
     tensor([[-1.1025e+00, -2.0554e-01, -1.4917e-01],
-            [-1.9429e-09, -9.1580e-01,  1.3878e-09]], dtype=torch.float64)
+            [-1.2490e-16, -9.1580e-01,  0.0000e+00]], dtype=torch.float64)

```

Comment:
Could this have been bugs at the time or something changed internally? Seems like params were fixed.

Reply:
This is a good question 🤔 Let me double check.

Reply:
@antalszava looks like this was a bug at the time :) I re-ran the original code examples, and get the same output as above.